.env.dist

```diff
@@ -27,6 +27,15 @@ MAIL_FROM_PREFIX=
 MAIL_USER=
 MAIL_PASS=
 
+# AZURE FUNCTION MAIL CONFIGURATION
+MAIL_SERVERLESS_URL=
+MAIL_SERVERLESS_KEY=
```

Comment:
@morgang-adappt 
what are the values for that?

Reply:
The PR has scaffolding for the Azure Function (an endpoint to proxy requests, variable keys added to `.env.dist`, and required changes made in `custom-environment-variables.js`, but the function is not stable enough.

I'd be happy to drop the commits until the function deployment if this is preferable. 

Additionally, how should we communicate these values once the Azure Function is complete? Do we just need to add the values to the App Service's environment variables?

Reply:
hey @morgang-adappt 

perhaps we can drop them yes, so we're sure about what we include

about the values, we'll add them to the azure service yes
once the function is complete, we can check together

Reply:
Hey @AntoineRelief ,

I've pushed a commit the removes all the discussed functionality.